code/game/objects/items/blueprints.dm

```diff
@@ -147,6 +147,9 @@ move an amendment</a> to the drawing.</p>
 			allthings.change_area(oldarea,newarea)
 	newarea.addSorted()
 
+	ghostteleportlocs += newarea.name
```

Comment:
what's your goal here, is it:
```list(name,area,name,area)```
or 
```list[name] = area```?

because if it's the latter, then the line below already accomplishes it all.